client/packages/cli/index.js

```diff
@@ -553,13 +546,51 @@ async function init() {
   }
 }
 
+async function importApp(pkgAndAuthInfo) {
+  console.log(
+    "Great! Grab your app id from: " +
+      chalk.blueBright.underline("https://instantdb.com/dash"),
+  );
+  const _id = await input({
+    message: "What's the app ID?",
+    required: true,
+  }).catch(() => null);
+  const id = _id?.trim();
+  if (!id) {
```

Comment:
Might be worth validating the input is a uuid? I wonder if someone may have a copy/paste error and would be nice if we can tell them can't find an app id?

(Maybe the error from `pull` hints this? I think right now the error message is terse?)

Reply:
Good point! I actually have a follow on PR which removes the manual prompt here entirely: 

https://github.com/instantdb/instant/pull/532

I also check now when app id is imported from env, whether it's a UUID or not, so the user should get a good error message 🫡